packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -24,6 +24,8 @@ export const messages = {
   channel_end_with_error: 'ended with error %s',
   channel_end: 'ended',
 
+  progress_notification_text: 'Running %s',
```

Comment:
![screen shot 2018-07-17 at 4 29 03 pm](https://user-images.githubusercontent.com/1639375/42849020-ccedb2f4-89de-11e8-8979-a242545cdf77.png)

Reply:
Does cancel kill the process?

Reply:
@amphro Yep, it kills the process.